src/HHbbVV/processors/bbVVSkimmer.py

```diff
@@ -1014,7 +1024,7 @@ def getVBFVars(
         # int list representing the number of passing vbf jets per event
         vbfVars[f"nGoodVBFJets"] = n_good_vbf_jets_sorted_pt.to_numpy()
 
-        adding_bdt_vars = True
+        adding_bdt_vars = False
```

Comment:
I changed this to false - can you confirm that we don't need these in the processor since they can be added in postprocessing?